src/main/java/com/g4s8/ghman/web/GHAuthRedirectionWrap.java

```diff
@@ -0,0 +1,47 @@
+package com.g4s8.ghman.web;
+
+import java.io.IOException;
+
+import org.apache.http.client.utils.URIBuilder;
+import org.cactoos.scalar.IoChecked;
+import org.takes.Request;
+import org.takes.Response;
+import org.takes.Take;
+import org.takes.facets.fork.FkWrap;
+import org.takes.facets.fork.Fork;
+import org.takes.misc.Opt;
+import org.takes.rs.RsRedirect;
+
+/**
+ * Github Authorization Redirection
+ * @since 1.0
+ * @todo #1:30min Refactor environment usage like APP_HOST, GH_CLIENT,
```

Comment:
@samihibrahim I think it's not going to play well with 0pdd to move the puzzle. I propose to leave it where it was. Especially now that it concerns at least two different classes (this one and `GHAuthorizationTake`).

What you should do though is add a comment in the corresponding github issue to explain it concerns those two files now.